comp110-worksheet-6/OxoBoard.cs

```diff
@@ -4,73 +4,97 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace comp110_worksheet_6
-{
-	public enum Mark { None, O, X };
+namespace comp110_worksheet_6 {
+    public enum Mark { None, O, X };
 
-	public class OxoBoard
-	{
-		// Constructor. Perform any necessary data initialisation here.
-		// Uncomment the optional parameters if attempting the stretch goal -- keep the default values to avoid breaking unit tests.
-		public OxoBoard(/* int width = 3, int height = 3, int inARow = 3 */)
-		{
-			throw new NotImplementedException("TODO: implement this function and then remove this exception");
-		}
+    public class OxoBoard {
+        // Constructor. Perform any necessary data initialisation here.
+        // Uncomment the optional parameters if attempting the stretch goal -- keep the default values to avoid breaking unit tests.
+        private Mark[,] boardSize;
```

Comment:
`boardSize` is a slightly confusing variable name -- it holds the contents of the board, not the size